### DISCUSSION TITLE
Default Active Tab

### DISCUSSION THREAD

**Discussion by adw259** [mdbootstrap/TW-Elements#1860] [Support from community] [answered]
I can use focus:bg-slate-200 to change the tab color when clicked but using active:bg-slate-200 to target the tab that is already visible on pageload does not work.
My code... 

<code>`
<a
                  href="#tabs-home"
                  class="block border-x-0 border-b-2 border-t-0 border-transparent px-7 pb-3.5 pt-4 text-xs font-medium  leading-tight text-neutral-500 hover:isolate hover:border-transparent bg-white focus:bg-slate-200 focus:isolate focus:border-transparent data-[te-nav-active]:border-primary data-[te-nav-active]:text-primary dark:text-neutral-400 dark:hover:bg-transparent dark:data-[te-nav-active]:border-primary-400 dark:data-[te-nav-active]:text-primary-400 rounded-tl-lg rounded-tr-lg"
                  data-te-toggle="pill"
                  data-te-target="#tabs-home"
                  data-te-nav-active
                  role="tab"
                  aria-controls="tabs-home"
                  aria-selected="true"
                  >Additional Information</a
                >
'
</code>

**Comment by juujisai** [comment, marked as answer]
To style the current / active tab use `data-[te-nav-active]:your-tw-class-here`. When a tab is active it receives  `data-te-nav-active` attribute
